beetsplug/convert.py

```diff
@@ -546,3 +526,45 @@ def _cleanup(self, task, session):
                 if os.path.isfile(path):
                     util.remove(path)
                 _temp_files.remove(path)
+
+    def _get_opts_and_config(self, opts):
```

Comment:
A quick docstring on this new method would be super helpful. (No need to even list all the options exhaustively, IMO—just giving the gist of what this does would be useful.)